notification/status.go

```diff
@@ -0,0 +1,83 @@
+package notification
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/influxdata/influxdb"
+)
+
+// StatusRule includes parametes of status rules.
+type StatusRule struct {
+	CurrentLevel  CheckLevel  `json:"currentLevel"`
+	PreviousLevel *CheckLevel `json:"previousLevel"`
```

Comment:
current level and previousLevel are meant to be levelRules which is a level and an operator rather than just the level itself https://github.com/influxdata/influxdb/blob/236322d86b557b4e74c62858db638a4ddead0a17/http/swagger.yml#L8891